pkg/platform/runtime/buildscript/buildexpression.go

```diff
@@ -0,0 +1,151 @@
+package buildscript
```

Comment:
I recommend reviewing this file last, as it ties together build scripts with the work-in-progress build expressions on the Platform side.